strawberryfields/apps/train/parametrization.py

```diff
@@ -0,0 +1,312 @@
+# Copyright 2020 Xanadu Quantum Technologies Inc.
```

Comment:
What about changing the name of the module? It won't be front facing anyway as the init above imports `VGBS` directly, but still good to choose a name that works.

I feel `parametrization` is quite long. What about `variational`? or just `var`?

Reply:
`var` is probably not good as it resembles variable. `variational` is an option but still long. How about `param`?

Reply:
Ok, I like it!